sympy/simplify/hyperexpand.py

```diff
@@ -2417,12 +2427,16 @@ def weight(expr, cond):
     return func0(z0)
 
 
-def hyperexpand(f, allow_hyper=False, rewrite='default'):
+def hyperexpand(f, allow_hyper=False, rewrite='default', place=None):
     """
     Expand hypergeometric functions. If allow_hyper is True, allow partial
     simplification (that is a result different from input,
     but still containing hypergeometric functions).
 
+    If a G-function has expansions both at zero and at infinity,
+    ``place`` can be set to ``0`` or ``zoo`` to indicate the
+    preferred choice.
```

Comment:
Ah, if you mean the default value of `place`, then I think that could be `None`, though `0` might not be a bad choice, either. I did not set the default to avoid having to edit broken tests in test_hyperexpand.py (and maybe in some other places where hyperexpand is used).